crates/chainql-core/src/log.rs

```diff
@@ -0,0 +1,15 @@
+#![allow(unused_imports)]
+
+#[cfg(all(feature = "log", feature = "log-tracing"))]
+compile_error!("Features `log-tracing` and `log` cannot be enabled together");
+
+#[cfg(all(not(feature = "log"), not(feature = "log-tracing")))]
+compile_error!("You should enable `log-tracing` or `log` feature");
+
+#[cfg(all(feature = "log-tracing", not(feature = "log")))]
+pub use tracing::{debug, info, warn, error};
+
+#[cfg(all(feature = "log", not(feature = "log-tracing")))]
+pub use log::{debug, info, warn, error};
+
+pub const LOG_TARGET: &str = "chainql";
```

Comment:
I don't think we necessarily need to have stable log target name (Honestly, polkadot-sdk in many cases too, it would've benefit from log granularity), the defaults (module name, e.g chainql_core::rebuild) are fine.